packages/flutter/lib/src/widgets/text_editing_intents.dart

```diff
@@ -69,6 +69,16 @@ class ExtendSelectionLeftByLineTextIntent extends Intent{
   const ExtendSelectionLeftByLineTextIntent();
 }
 
+/// An [Intent] to extend the selection left past the nearest word, collapsing
```

Comment:
Looks like the method seems to extend the selection towards the start of the text, instead of moving left. Does that match macOS's behavior?

Reply:
in Pages in RTL text pressing <kbd><-</kdb> the cursor moves towards the end of the text.